### ISSUE TITLE
cy.type() on a DIV

### ISSUE THREAD

**Issue by souljacker** [cypress-io/cypress#1686]
I have an app which is a text editor and the DOM is formatted using regular elements (h1, div, span) and they become actionable when clicked and the user can type as in a regular textarea  or input . Cypress won't let me type on it, though. 

I tried using force, with no success.

Is there any way to make it work or I'm stuck with Dusk? :(

EDIT:

I tried the following solution but it only types the first character on the desired field and the rest switches to another one.

Here's my DOM and how it should work 
https://imgur.com/fCjZLRK

Here's how it's happening with Cypress
https://imgur.com/Oynzlrd

And here's my current code

```js
cy.get('.editor-title').find('[contenteditable]').click().type('An unexpected party', {
      force: true,
    });
```

**Comment by jennifer-shehane** [comment]
Cypress allows typing into elements with a `contenteditable` attribute - as outlined [here](https://github.com/cypress-io/cypress/issues/1650) and it does look like it is typing at least one character without erroring. 

I suspect that the library you are using is listening to or triggering events that Cypress is not handling correctly. Could you provide a reproducible example?

Also, as workarounds, try playing with the `delay` option of the type or using `trigger` to set the value directly in the div.

**Comment by Bkucera** [comment]
This should be fixed by PR #2016
